test/packages/EABase/include/Common/EABase/config/eacompiler.h

```diff
@@ -726,9 +726,9 @@
 			// Extern template is supported.
 		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__EDG_VERSION__) && (__EDG_VERSION__ >= 401)  // EDG 4.1+.
 			// supported.
-		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__APPLE__) && (EA_COMPILER_VERSION >= 401)
+		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__apple_build_version__) && (EA_COMPILER_VERSION >= 401)
```

Comment:
The problem is clang3.9 is not AppleClang. It's just plain old "Clang" and so 3.9 looks like a really old version of AppleClang because 3.9 < 4.0.1 but really it should be hit the non-apple clang comparison (where any version of "Clang" is ok).

See this snippet:
		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && defined(__apple_build_version__) && (EA_COMPILER_VERSION >= 401)
			// Extern template is supported.
		#elif defined(EA_COMPILER_CPP11_ENABLED) && defined(__clang__) && !defined(__apple_build_version__)             // Clang other than Apple's Clang
			// Extern template is supported.